### ISSUE TITLE
Rendering Issue Rustgrad

### ISSUE THREAD

**Issue by Whatscheiser** [zturtleman/spearmint#331]
I've been downloading quite a few maps. After a few days of going through quite a few levels I'm fairly confident when I say this is probably the last render issue I'll need to post.

That said I recorded another video for this one: https://www.youtube.com/watch?v=yECbtK-L9w0 
In Rustgrad by Hipshot I'm having an issue with these vertical columns rendering in properly. My best stab at this one, is that these columns sit behind some kind of noclipping brush so players don't get hung up on them during deathmatch. My guess is the noclip is being rendered somehow and causing a conflict 

Again, just guessing. This issue occurs using both the opengl1 and opengl2 renderer.

**Comment by zturtleman** [comment]
Behind the wood column is a decal shader with `polygonOffset`. I changed the polygon offset values in Spearmint d85c2092e6ac5d975b0045579dec64e2301f8a98 and in this situation causes the decal to offset in front of the wood column or Z-fight with it. The values are passed to OpenGL's [glPolygonOffset() function](https://docs.gl/gl2/glPolygonOffset) that I don't really understand.

Changing to the vanilla Quake 3 / ioquake3 values solves this issue.

Spearmint
```
    r_offsetunits = "-600"
    r_offsetfactor = "0"
```

ioquake3
```
    r_offsetunits = "-2"
    r_offsetfactor = "-1"
```

**Comment by Whatscheiser** [comment]
Can confirm setting the ioquake3 values resolves this issue as well. I should stop suggesting theories I'm 0 for 2 today. LOL.

In order to set this I need to load Rustgrad through the console as a devmap. Otherwise I get feedback stating the value of the setting is cheat protected. Is there somewhere I can set these values so they stay applied. If possible I like to avoid loading the map through the console.

Thanks again for the quick response by the way.

**Comment by Whatscheiser** [comment]
I guess I'll just use the console :p

No worries, thank you for responding to my questions. Like I said before I do appreciate it. Would you like me to close out these issues I posted in here?

**Comment by zturtleman** [comment]
No, leave them open. They're real issues and may be fixed eventually. I'm just not really working on Spearmint at this point. Thanks for reporting them.

**Comment by Whatscheiser** [comment]
You're welcome. One last thing, and I'm sorry to go off topic but this is more a curiosity than a thing that needs its own thread...
Running Quake 3 under Spearmint in 21:9 resolutions... during the campaign, at the end of a match, you were able to get the podium with the player and bots to display the correct distance away from the screen. I'm wondering if that is something that can be tweaked in the console or if that was some handy work you baked into Spearmint?

**Comment by zturtleman** [comment]
I don't remember doing anything with it and code looks unchanged. Field of view (cg_fov cvar) might affect how it appears. Spearmint automatically converts cg_fov cvar value (assumed 4:3 aspect) to the current aspect (or set cg_fovAspectAdjust 0 to disable this). You can use https://casualhacks.net/Source-FOV-calculator.html "What's my real FOV?" to get "Actual FOV" to set cg_fov to in other Quake 3 engines.
